src/wasm-lib/kcl/src/executor.rs

```diff
@@ -1980,9 +1980,52 @@ impl From<crate::settings::types::ModelingSettings> for ExecutorSettings {
     }
 }
 
+/// Create a new zoo api client.
+#[cfg(not(target_arch = "wasm32"))]
+pub fn new_zoo_client(token: Option<String>, engine_addr: Option<String>) -> Result<kittycad::Client> {
+    let user_agent = concat!(env!("CARGO_PKG_NAME"), ".rs/", env!("CARGO_PKG_VERSION"),);
+    let http_client = reqwest::Client::builder()
+        .user_agent(user_agent)
+        // For file conversions we need this to be long.
+        .timeout(std::time::Duration::from_secs(600))
+        .connect_timeout(std::time::Duration::from_secs(60));
+    let ws_client = reqwest::Client::builder()
+        .user_agent(user_agent)
+        // For file conversions we need this to be long.
+        .timeout(std::time::Duration::from_secs(600))
+        .connect_timeout(std::time::Duration::from_secs(60))
+        .connection_verbose(true)
+        .tcp_keepalive(std::time::Duration::from_secs(600))
+        .http1_only();
+
+    let token = if let Some(token) = token {
+        token
+    } else if let Ok(token) = std::env::var("KITTYCAD_API_TOKEN") {
+        token
+    } else if let Ok(token) = std::env::var("ZOO_API_TOKEN") {
+        token
+    } else {
+        return Err(anyhow::anyhow!(
+            "No API token found in environment variables. Use KITTYCAD_API_TOKEN or ZOO_API_TOKEN"
+        ));
+    };
+
+    // Create the client.
+    let mut client = kittycad::Client::new_from_reqwest(token, http_client, ws_client);
+    // Set an engine address if it's set.
+    if let Some(addr) = engine_addr {
+        client.set_base_url(addr);
+    } else if let Ok(addr) = std::env::var("ZOO_HOST") {
+        client.set_base_url(addr);
+    } else if let Ok(addr) = std::env::var("KITTYCAD_HOST") {
+        client.set_base_url(addr);
```

Comment:
Same here - I can imagine someone tries KITTYCAD_HOST somewhere when ZOO_HOST is already set and they're like "wtf"

Reply:
okay fine ill do it

Reply:
Alt. is completely remove one IMO

Reply:
done, i ignore it if they are the same

Reply:
Perfect, thank you - I can just already hear someone being like "wtf what's going on here" because env vars are particularly global and could be set _anywhere_ in a system - any other case I wouldnt have cared